### DISCUSSION TITLE
[Feature] NodeJS Agent Kafka  Support

### DISCUSSION THREAD

**Discussion by muratkarakas** [apache/skywalking#12570] [Ideas]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Current NodeJS agent does not support KafkaJS(or any Kafka lib). This feature would give move visibility for the applications that heavily use Kafka.Is it possible to support this like RabbitMQ?

### Related issues

https://github.com/apache/skywalking/discussions/12563

### Are you willing to submit a pull request to implement this on your own?

- [ ] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
Are you going to contribute this? Our issues are kept when we have owner.

Others are in discussion channel only.

SkyWalking agents rely on contributions.

**Comment by wu-sheng** [comment]
> Is it possible to support this like RabbitMQ?

From tech perspective, yes. But we don't have that from server side.

You could do that in your own fork. OAP is easy to expand.
